staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -760,6 +760,23 @@ func TestApplyPruneObjectsWithAllowlist(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// Create Namespace that can be pruned
+	// Deprecated: kubectl apply will no longer prune non-namespaced resources by default when used with the --namespace flag in a future releas
```

Comment:
Won't the test for the case where the namespace flag is NOT provided still need this? If so, I don't think we need the deprecation message.

Reply:
Removed.